tasks/Task.cpp

```diff
@@ -25,6 +25,17 @@ Task::~Task()
 
 bool Task::configureHook()
 {
+    /**
+     * Apply the camera parameters
+     */
+    underwater_camera_simulation::CameraParams params = _camera_params.get();
+
+    /**
+     * set the viewport size the scene size of the camera
+     */
+    _viewport_width.set((params.width <= 0) ? 800 : params.width);
```

Comment:
`<= 0` ???? Don't overdo things, return an error if the width/height is not valid, there is no "natural" resolution for this component (unlike a camera which has a native resolution ... I guess)